lib/ecto/repo/supervisor.ex

```diff
@@ -169,6 +169,12 @@ defmodule Ecto.Repo.Supervisor do
   def init({name, repo, otp_app, adapter, opts}) do
     case runtime_config(:supervisor, repo, otp_app, opts) do
       {:ok, opts} ->
+        :telemetry.execute(
+          [:ecto, :repo, :start],
```

Comment:
I wouldn't use start in here because people may think this is a trace and that eventually we will have a stop/exception. What do you think about init or boot?

Reply:
Agreed, I'll change to `init` to match the action that is actually happening